bundle/edu.gemini.spdb.reports.collection/src/test/java/edu/gemini/dbTools/ephemeris/EphemerisFilesTest.scala

```diff
@@ -46,9 +46,8 @@ object EphemerisFilesTest extends Specification with ScalaCheck with Arbitraries
   def testRight[T](makeAction: EphemerisFiles => TryExport[T]): T =
     execTest(makeAction) {
       case -\/(err) =>
-        val (msg, ex) = err.report
-        log.log(Level.WARNING, msg, ex.orNull)
-        throw ex.getOrElse(new RuntimeException())
+        log.log(Level.WARNING, err.message, err.exception.orNull)
+        throw err.exception | new RuntimeException()
```

Comment:
It's just `getOrElse` added by `scalaz.syntax.std.OptionOps`.  The idea here is just to make sure that the test throws some exception so that it fails if the result is a left.